admin/controllers/ping.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const { getCommitId, getBuildNumber } = require('../helpers/healthcheck')
+
+/* Health check */
+async function getPing (req, res) {
+  // get build number from /build.txt
+  // get git commit from /commit.txt
+  let buildNumber = 'NOT FOUND'
+  let commitId = 'NOT FOUND'
+  try {
+    buildNumber = await getBuildNumber()
+  } catch (error) {
+
+  }
```

Comment:
Your code is fine, but I quite like:

```suggestion
  } catch (ignore) {}
```